ReactiveCocoaTests/Swift/SignalLifetimeSpec.swift

```diff
@@ -26,37 +26,41 @@ class SignalLifetimeSpec: QuickSpec {
 				expect(signal).to(beNil())
 			}
 
-			it("should deallocate even if it has an observer") {
+			it("should deallocate if it does not have any observers") {
```

Comment:
No, I mean when the signal never gets any observers, but it's temporarily retained.